lua/telescope/pickers.lua

```diff
@@ -326,6 +326,9 @@ function Picker:find()
 
   self.original_win_id = a.nvim_get_current_win()
 
+  -- Use autocmd run it before create Telescope window
+  vim.cmd'do User Telescope'
```

Comment:
Can we call it `TelescopeFindPre`? Since it will be run before every find?

If that's fine, I'd like to see it as `vim.cmd [[doautocmd User TelescopeFindPre]]`

If you do that, then we can merge is fine with me.